packages/components/calendar/src/calendar.stories.ts

```diff
@@ -49,12 +81,31 @@ export default {
       return value instanceof Date ? value : new Date(value);
     };
 
+    const onSelectDate = (event: CustomEvent<Date>) => {
+      updateArgs({ selected: new Date(event.detail).getTime() }); //needs to be set to the 'time' otherwise Storybook chokes on the date format 🤷
+    };
+
     return html`
       <sl-calendar
```

Comment:
It looks like the space between week days and days is too small comparing to Figma?

In the component:

<img width="318" height="130" alt="Image" src="https://github.com/user-attachments/assets/8f841b04-c235-4ceb-83ce-b35cbdc636de" />

In Figma:

<img width="598" height="223" alt="Image" src="https://github.com/user-attachments/assets/3c32b2f3-aa40-4b81-9704-9519db72e8be" />